sql/telemetry_derived/clients_histogram_aggregates_v1/query.sql

```diff
@@ -1,3 +1,37 @@
+CREATE TEMP FUNCTION udf_clear_histogram_aggregates(aggs ANY TYPE)
+  RETURNS ARRAY<STRUCT<
+    first_bucket INT64,
+    last_bucket INT64,
+    num_buckets INT64,
+    latest_version INT64,
+    metric STRING,
+    metric_type STRING,
+    key STRING,
+    process STRING,
+    agg_type STRING,
+    old_aggregates ARRAY<STRUCT<key STRING, value INT64>>,
+    new_aggregates ARRAY<STRUCT<key STRING, value INT64>>>> AS (
+  (
+    WITH unnested AS
+        (SELECT * REPLACE(CAST(NULL AS ARRAY<STRUCT<key STRING, value INT64>>) AS old_aggregates)
+        FROM UNNEST(aggs))
+
+    SELECT ARRAY_AGG((
+        first_bucket,
+        last_bucket,
+        num_buckets,
+        latest_version,
+        metric,
+        metric_type,
+        key,
+        process,
+        agg_type,
+        old_aggregates,
+        new_aggregates))
```

Comment:
The purpose of `old_aggregates` is to provide the old values for histograms that need to be subtracted from the current bucket counts. `old_aggregates`, when it's not `NULL` is a copy of the `new_aggregates` value from the previous day. This copy is created so that it's more easily accessible from the bucket counts query.